app/src/main/res/values/strings.xml

```diff
@@ -1,10 +1,11 @@
 <resources>
-    <string name="common_ok">OK</string>
-    <string name="common_yes">Yes</string>
-    <string name="common_no">No</string>
-    <string name="common_cancel">Cancel</string>
-    <string name="common_delete">Delete</string>
-    <string name="common_remove">Remove</string>
+    <!-- Common strings, uncomment if you want to use them -->
+    <!--<string name="common_ok">OK</string>-->
+    <!--<string name="common_yes">Yes</string>-->
+    <!--<string name="common_no">No</string>-->
+    <!--<string name="common_cancel">Cancel</string>-->
+    <!--<string name="common_delete">Delete</string>-->
+    <!--<string name="common_remove">Remove</string>-->
 
```

Comment:
My general philosophy with Skeleton is to cover the common cases even if it means providing more than what each specific project needs. It's much easier to delete something than to create. People can run the analysis at the beginning and delete the things they don't want/use